nodejs/week3/app.js

```diff
@@ -0,0 +1,55 @@
+const knex = require("knex")({
+  client: "mysql2",
+  connection: {
+    host: process.env.DB_HOST || "127.0.0.1",
+    port: process.env.DB_PORT || 3306,
+    user: process.env.DB_USER || "root",
+    password: process.env.DB_PASSWORD || "123456789",
```

Comment:
Just a gentle reminder to use a .env file for sensitive data like passwords. It is okay in this case.